<?xml version="1.0"?>
<launch>
    <!-- Тип автопилота -->
    <!-- 0 - PX4 -->
    <!-- 1 - ArduPilot -->
    <arg name="ap_type" default="0" />
    <!-- Высота взлета над землей в метрах -->
    <arg name="altitude" default="1.5" />
    <!-- Использовать визуальную одометрию или нет -->
    <arg name="use_vision_odometry" default="0" />
    <!-- Время удержания позиции в секундах: целое положительное число -->
    <arg name="hover_time" default="60" />

    <node name="$(anon position_hold_test)" pkg="flight_tests" type="position_hold_test.py" output="screen">
        <param name="altitude" value="$(arg altitude)" />
        <param name="autopilot_type" value="$(arg ap_type)" type="bool"/>
        <param name="use_vision_odometry" value="$(arg use_vision_odometry)" type="bool"/>
        <param name="hover_time" value="$(arg hover_time)"/>
    </node>
</launch>
